doc/content/devices/mac-settings/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "MAC Settings"
+description: ""
+---
+
+This section provides guidelines for configuring MAC settings for end devices from the CLI.
+
+<!--more-->
+
+{{< cli-only >}}
+
+### MAC Settings and MAC State
+
+`mac-settings` on {{% tts %}} are configurable per end device. To configure persistent MAC settings, make changes to `mac-settings.desired_<parameter>`. Updates to `mac-settings.desired_<parameter>` take effect on device creation, after OTAA join or ABP FCnt reset, ResetInd, or after MAC state reset.
```

Comment:
```suggestion
Various MAC settings on {{% tts %}} are configurable per end device. To configure persistent MAC settings, make changes to `mac-settings.desired_<parameter>`. Updates to `mac-settings.desired_<parameter>` take effect on device creation, after OTAA join or ABP FCnt reset, ResetInd, or after MAC state reset.
```

Reply:
This should mention `mac-settings.current_<parameter>` as well - those are generally necessary when device is not fully compliant with the spec, but also in other cases